ansible/pbTestScripts/testJDKWin.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
 
-# Script to execute System tests on the previously built JDK.
+# Relocate Built JDKS
+mv /cygdrive/c/tmp/workspace/build/src/build/*/images/jdk* c:/tmp
 
-mv /cygdrive/c/tmp/workspace/build/src/build/*/images/jdk* $HOME
-# Ensures to set it to the JDK, not JRE or different images
-export TEST_JDK_HOME=C:/cygwin64$(find ~ -maxdepth 1 -type d -name "*jdk*"|grep -v ".*jre"| grep -v ".*-image")
+# Remove Redundant Images
+rm -rf /cygdrive/c/tmp/*-debug-image
```

Comment:
i thought the file is called *debugimage* and *testimage* without "-"

Reply:
It appears on windows this is definitely what the VPC build produces, only the actual JDK is needed, so I remove the redundant bits to save a bit of disk space, as these windows vagrant boxes are starting to get quite large.